solutionDraw3d.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import numpy as np
```

Comment:
Fun random fact: I overlapped with one of the creators of NumPy, and also of IronPython, Jim Hugunin, at MIT in grad school!

Reply:
Wow, that is cool :).